invenio_stats/queries.py

```diff
@@ -29,7 +30,7 @@ def __init__(self, query_name, index, client=None, *args, **kwargs):
         :param client: elasticsearch client used to query.
         """
         super(ESQuery, self).__init__()
-        self.index = index
+        self.index = prefix_index(index)
```

Comment:
Hm, considering that this was done already in `ext.py` I'm suspicious of what actually happens (i.e. one of the two doesn't actually prefix anything)

Reply:
I just ran the tests and they are passing if I remove the `prefix_index` from `ext.py`. It might be that this was not executed during the tests. Anyway, I think we can remove it.